book/projects/project_initialization.md

```diff
@@ -16,7 +16,7 @@ Before getting started, you'll need a name for your project. A team exploring sn
 We have created a GitHub template that is pre-populated with directories and a README layout the we find works well with most teams.
 
 We encourage all teams to create a new GitHub repository within the {{hackweek}} Organization, even if it is just to point to an already-existing repository or package your team is contributing to for their project.
-1. Go to: {{ '[https://github.com/`{hw}`](https://github.com/{hw})'.format(hw=hackweek) }}
+1. Go to: {{ '[`{github_org_url}`]({github_org_url})'.format(hw=hackweek) }}
```

Comment:
Looks like this is causing trouble.
If link name and target are the same, you could try this syntax:
https://www.markdownguide.org/basic-syntax#urls-and-email-addresses

Reply:
It was plugging in "hw", which is the event name and has a space in it (and thus is an invalid url). I changed it to use `github_org_url`. Must have missed this when we previously separated out the link vs name in the config.

Reply:
So we should flag this to fix in the template - do we have a spot going for this here or in the template?

Reply:
There is also a pending update to JupyterBook we haven't done (which might improve handling of string interpolation) and I suggest to log an issue